modules/vault-secrets.nix

```diff
@@ -233,6 +242,8 @@ in
           ${extraScript}
         '' + optionalString (user != null) ''
           chown -R "${user}:nobody" "${secretsPath}"
+        '' + optionalString (group != null) ''
```

Comment:
Just `chown -R "${user}:${group}" "${secretsPath}"`

Reply:
Simplified by removing the null case.